conf/etc/nginx/sites-available/default

```diff
@@ -23,6 +23,7 @@ server {
 
 	root /opt/piecewise_web;
 	index index.html index.htm;
+	sendfile off;
```

Comment:
This came from troubleshooting issues with the vbox file system without the vbox guest additions iso installed on the host system. I'll retest without it to confirm whether it's needed.